src/utils/data_handling.py

```diff
@@ -21,4 +21,4 @@ def states_to_df(states: List[PropagatedOutput]) -> pd.DataFrame:
 def df_to_csv(dataframe: pd.DataFrame, path: Optional[Union[str, Path]] = None):
     if path is None:
         path = SIM_ROOT / "runs"
-    dataframe.to_csv(f"{path}/cislunarsim-{time.time()}.csv")
+    dataframe.to_csv(f"{path}/cislunarsim-{int(time.time())}.csv")
```

Comment:
Hm, should we be adding something here that differentiates the csv more? Since times could potentially overlap which each other?

Reply:
Yeah, some sort of ID would be good. The times are only being truncated to the nearest second, and we don't (yet) have multiple sim runs occurring within a second. I don't think a randomly generated ID would be helpful though, because you'd want to know something useful about the run from its filename.

Reply:
Ok! Maybe something to look at in the future, but not really an issue right now.